### PR TITLE
Adapt carrier tests/CI for fluo

### PR DESCRIPTION
**Pull request by flaviodsr**

### CODE REVIEW

File: acceptance/acceptance_suite_test.go [hunk 1]
```diff
@@ -100,7 +100,7 @@ func createCluster() {
 		panic("Couldn't find k3d in PATH: " + err.Error())
 	}
 
-	_, err := RunProc("k3d cluster create "+name, nodeTmpDir, false)
+	_, err := RunProc("k3d cluster create --k3s-server-arg '--kubelet-arg=eviction-hard=imagefs.available<1%,nodefs.available<1%' --k3s-server-arg '--kubelet-arg=eviction-minimum-reclaim=imagefs.available=1%,nodefs.available=1%' "+name, nodeTmpDir, false)
```
Comment (jsuchome) [inline acceptance/acceptance_suite_test.go:103]: Why is this needed? Does not seem fluo related

Comment (flaviodsr) [reply acceptance/acceptance_suite_test.go:103]: The reason the job was failing (as shown on the meeting) was because by default kubernetes has the disk toleration set to 20% which means if a node has less than 20% available disk space it will evict the pods from that node. This is what was happening just after the job that builds the image.

This change sets the toleration to 1% so the pods does not get evicted.

**Review by jsuchome — APPROVED** [review]
